examples/project_cleanup/function_source/README.md

```diff
@@ -0,0 +1,20 @@
+# Project Cleanup Utility
+
+This is a simple utility that scans a GCP organization for projects matching certain criteria, and enqueues such projects for deletion. Currently supported criteria are the combination of:
+
+- **Age:** Only projects older than the configured age, in hours, will be marked for deletion.
+- **Key-Value Pair:** Only projects whose labels contain the provided key-value pair will be marked for deletion.
```

Comment:
I think it's a little unclear if a project has to match either age or KV pair, or both. As in, are the results for each AND'd together, or OR'd.

Reply:
+1 need precision on "OR" or "AND"

Reply:
@Jberlinsky Based on your initial comment, it will be an "AND", correct ?